ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/ImageCve/ImageCveSummaryCards.tsx

```diff
@@ -24,10 +24,18 @@ export type ImageCveSeveritySummary = {
 export const imageCveSeveritySummaryFragment = gql`
     fragment ImageCVESeveritySummary on ImageCVECore {
         affectedImageCountBySeverity {
-            critical
-            important
-            moderate
-            low
+            critical {
```

Comment:
Don't you need fixable info for "CVEs by status" card? Or that will be a separate PR?

Reply:
That's separate, this fragment is only used for the ImageCVE page, which doesn't show the summary card for "by status". I've got a follow up PR that is going to consolidate these a little better as well with the addition of the deployment page.